docs/gulpfile.js

```diff
@@ -14,343 +14,427 @@
-        // we'll simply copy over instead of creating symlinks
-        // .pipe(symlink('components/modules/ROOT/pages/', {
-        //     relativeSymlinks: true
-        // }));
-        // uncomment above .pipe() and remove the .pipe() below
-        // when antora#188 is resolved
-        .pipe(insertSourceAttribute())
-        .pipe(dest('components/modules/ROOT/pages/'));
+function createComponentSymlinks () {
+  return (
+    src([
+      '../core/camel-base/src/main/docs/*-component.adoc',
+      '../components/{*,*/*}/src/main/docs/*-component.adoc',
+      '../components/{*,*/*}/src/main/docs/*-summary.adoc',
+    ])
+      .pipe(
+        map((file, done) => {
+          // this flattens the output to just .../pages/....adoc
+          // instead of .../pages/camel-.../src/main/docs/....adoc
+          file.base = path.dirname(file.path)
+          done(null, file)
+        })
+      )
+      // Antora disabled symlinks, there is an issue open
+      // https://gitlab.com/antora/antora/issues/188
+      // to reinstate symlink support, until that's resolved
+      // we'll simply copy over instead of creating symlinks
+      // .pipe(symlink('components/modules/ROOT/pages/', {
+      //     relativeSymlinks: true
+      // }));
+      // uncomment above .pipe() and remove the .pipe() below
+      // when antora#188 is resolved
+      .pipe(insertSourceAttribute())
+      .pipe(dest('components/modules/ROOT/pages/'))
+  )
 }
 
-function createComponentOthersSymlinks() {
-    const f = filter(['**','!**/*-language.adoc', '!**/*-dataformat.adoc', '!**/*-component.adoc', '!**/*-summary.adoc'])
-    return src(['../core/camel-base/src/main/docs/*.adoc','../core/camel-main/src/main/docs/*.adoc','../components/{*,*/*}/src/main/docs/*.adoc'])
-        .pipe(f)
-        .pipe(map((file, done) => {
-            // this flattens the output to just .../pages/....adoc
-            // instead of .../pages/camel-.../src/main/docs/....adoc
-            file.base = path.dirname(file.path);
-            done(null, file);
-        }))
-        // Antora disabled symlinks, there is an issue open
-        // https://gitlab.com/antora/antora/issues/188
-        // to reinstate symlink support, until that's resolved
-        // we'll simply copy over instead of creating symlinks
-        // .pipe(symlink('components/modules/ROOT/pages/', {
-        //     relativeSymlinks: true
-        // }));
-        // uncomment above .pipe() and remove the .pipe() below
-        // when antora#188 is resolved
-        .pipe(insertSourceAttribute())
-        .pipe(dest('components/modules/others/pages/'));
+function createComponentOthersSymlinks () {
+  const f = filter([
+    '**',
+    '!**/*-language.adoc',
+    '!**/*-dataformat.adoc',
+    '!**/*-component.adoc',
+    '!**/*-summary.adoc',
+  ])
+  return (
+    src(['../core/camel-base/src/main/docs/*.adoc', '../components/{*,*/*}/src/main/docs/*.adoc'])
```

Comment:
Seems that `'../core/camel-main/src/main/docs/*.adoc'` is missing here.

Reply:
Yes, that's the merge conflict.

Reply:
I have rebased my branch with the master and it's up to date with the master branch. I am not sure about how this difference has come.

Reply:
Claus added a commit to gulpfile.js after your initial run of lint[-fix].  His change isn't in your latest commit.  I think you should discard your second commit, make your your branch is up to date with master (in case someone else changed something :-), run lint-fix, and make any manual changes needed, commit the result, and force-push.  I hope we'll be able to commit then :-)

Reply:
Latest commit includes the camel-main clause.